### PR TITLE
Work in progress to create initial disease list

### PR DESCRIPTION
**Pull request by matentzn**
makefile and associated scripts, and filtering documentations

### CODE REVIEW

File: docs/filtering.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+
+_List of current heurstics_ for the MATRIX disease list:
+
+- INCLUDE all [leaf classes](#leaf-filter)
+- INCLUDE all [direct parents of leaf classes](#filter-leaf) IF
+    - they are [mapped to ICD10 categories](#filter-icd10) OR
+    - they [correspond to OMIM diseases](#filter-omim) OR
+    - they [correspond to descendants of OMIM Phenotypic Series (OMIMPS)](#filter-omimps-descendants)
+    - they [correspond to Orphanet Disorders](#filter-orphanet-disorder)
+    - they [correspond to Orphanet Subtypes](#filter-orphanet-subtypes)
```
Comment (matentzn) [inline docs/filtering.md:15]: From @sabrinatoro:

> while looking at examples, I realize that we are excluding terms that should be included. 
1. For example, terms that are in orphanet-disorder subset AND omimPS are excluded. They should be included.
2. MONDO:0009723 is excluded, but it is an orphanet disorder and has an omim id (though not shown in the spreadsheet)
3. MONDO:0100151 is excluded, but it has an omim id (though not shown in the spreadsheet)

Comment (elliottsharp) [reply docs/filtering.md:15]: considering some ICD-9 codes are included, are these also tracked alongside ICD-10CM?

Comment (matentzn) [reply docs/filtering.md:15]: No, but I am not so convinced at the moment of the quality of the ICD9 mappings. But we can consider it for a future iteration!

Comment (matentzn) [reply docs/filtering.md:14]: This is now solved by having orphanet disorder and icd 10 category in the target set

**Pull request merged**